catalog/build_ci/B002.md

```diff
@@ -2,9 +2,8 @@
 
 | **Code**           | **B002** |
 | :--                | :--      |
-| **Title**          | Enforcing a Specific Code Style for Main Programming Languages. |
-| **Description**    | There should be an agreed code style for each of main programming languages. Also, some procedures prevent merging inconsistent code with the code style. |
+| **Title**          | A specific code style is enforced for main programming languages |
```

Comment:
According to the following address, you may use "reinforced"
https://english.stackexchange.com/questions/502559/how-can-enforce-and-reinforce-have-slightly-different-spelling-and-still-be-vali

Reply:
Based on that link, I think yet it should be "enforced"; since the item actually aims for obedience against a unified code style.  
Let me know if you disagree.